### ISSUE TITLE
The effect is lower than expected

### ISSUE THREAD

**Issue by qianyeqiang** [zhaoweicai/mscnn#62]
Dear author
I test the KITTI vehicle training dataset using the mscnn-8s-768-trainval-pretrained, which is download as README. Then result is evaluated.
The result is 77% (moderate), which is far below the result in KITTI list. Since the training set and the test set are the same, the result should be close to 100% from my experience.
So, what the problem? Thank you very much.
@zhaoweicai

**Comment by qianyeqiang** [comment]
I have find the problem, I change the detection code incorrectly. I will close the issue.
